cli/lsp/documents.rs

```diff
@@ -1151,7 +1156,9 @@ impl Documents {
         &specifier,
         self.get_npm_resolver(),
       )
-    }
+    };
+    self.performance.measure(mark);
```

Comment:
This will conflate getting an already included document and one that needs to be fs-read in `FileSystemDocs::refresh_document()`. Is that okay?

Reply:
Probably not. What would you suggest to do here?